.github/workflows/reusable-translations.yml

```diff
@@ -114,7 +135,6 @@ jobs:
         uses: shivammathur/setup-php@9e72090525849c5e82e596468b86eb55e9cc5401 # v2.32.0
         with:
           php-version: "8.0"
-          tools: composer
```

Comment:
@circlecube @wpscholar I think this may have been causing the problems with the Composer script not running properly as seen in the [Bluehost plugin repository](https://github.com/newfold-labs/wp-plugin-bluehost/actions/runs/16006919987/job/45166256207#step:7:40).